package_schemas/country_estimates/2_country_estimates_22.json

```diff
@@ -12,10 +12,10 @@
             "form_snippet": "text.html",
             "validators": "autogenerate",
             "classes": ["hidden"],
-            "template": "{} Country Estimates 2022",
+            "template": "{} HIV Estimates 2022",
             "template_args": ["geo-location", "year"],
```

Comment:
```
"template_args": ["geo-location", "year"],
```
I think that this part is a hidden bug. Only the "geo-location" is used to format the template. It doesn't cause any issues only due to poor error handling of the autogenerate validator.

Could you check if changing those values to:
```
            "template": "{geo-location} Country Estimates 2022",
            "template_args": ["geo-location"]
```
still works?

If you don't have time to do that then let's forget about it for now.